camel-kit-core/src/main/resources/skills/camel-ship/SKILL.md

```
@@ -0,0 +1,99 @@
+|---|---|---|
+| `[input-file]` | none | Requirements document, design spec, or brainstorm notes |
+| `--ask` | `smart` | Oversight level: `always`, `smart`, or `never` |
+| `--resume` | false | Continue from `.camel-kit/ship-state.json` |
+| `--start-from <stage>` | none | Skip to stage: `brainstorm`, `plan`, `execute`, `verify` |
+| `--create-pr` | false | Auto-create GitHub PR on success |
+
+---
+
+## Pipeline
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)